admin/osx/CMakeLists.txt

```diff
@@ -11,7 +11,7 @@ else()
   set(MAC_INSTALLER_DO_CUSTOM_BACKGROUND "false")
 endif()
 
-find_package(Qt5 5.6 COMPONENTS Core REQUIRED)
+find_package(Qt5 ${OC_QT_VERSION} COMPONENTS Core REQUIRED)
```

Comment:
Do we use Qt at all in here?

Reply:
Doesn't look like it. Do you want me to remove it?

Reply:
yes